Containerfile

```diff
@@ -17,33 +18,29 @@ COPY ./scripts ./scripts
 RUN dnf install -y graphviz python-devel pcre-devel
 RUN pip install tox
 
-RUN if [ -z "${RHOSO_DOCS_GIT_URL}" ]; then \
+RUN if [ "$BUILD_UPSTREAM_DOCS" = "true" ]; then \
         ./scripts/get_openstack_plaintext_docs.sh; \
     fi
 
 RUN if [ ! -z "${RHOSO_DOCS_GIT_URL}" ]; then \
-        ./scripts/get_rhoso_plaintext_docs.sh; \
+        OUTPUT_DIR_NAME=rhoso-docs-plaintext ./scripts/get_rhoso_plaintext_docs.sh; \
     fi
 
-RUN if [ -z "${RHOSO_DOCS_GIT_URL}" ]; then \
-        python ./scripts/generate_embeddings_openstack.py \
-            --output ./vector_db/ \
-            --openstack-folder openstack-docs-plaintext/ \
-            --model-dir embeddings_model \
-            --model-name ${EMBEDDING_MODEL} \
-            --index ${INDEX_NAME} \
-            --workers ${NUM_WORKERS}; \
-    fi
+RUN CMD="python ./scripts/generate_embeddings_openstack.py"; \
```

Comment:
(non-blocking nit) It took me a moment to figure out why you had changed this. It looks a bit awkward and I wonder if this would work? (untested, but constructed through careful copy/paste ;)

```
RUN if [ "$BUILD_UPSTREAM_DOCS" = "true" ]; then \
        FOLDER_ARG="--folder openstack-docs-plaintext/"; \
    fi && \
    if [ ! -z "${RHOSO_DOCS_GIT_URL}" ]; then \
        FOLDER_ARG="$FOLDER_ARG --rhoso-folder rhoso-docs-plaintext"; \
    fi && \
    python ./scripts/generate_embeddings_openstack.py \
    --output ./vector_db/ \
    --openstack-folder openstack-docs-plaintext/ \
    --model-dir embeddings_model \
    --model-name ${EMBEDDING_MODEL} \
    --index ${INDEX_NAME} \
    --workers ${NUM_WORKERS} \
    ${FOLDER_ARG}
```

Reply:
I will give it a try, hope it works cause looks much better than the committed one

Reply:
Seems to work! Just had to remove the "--openstack-folder" from the example as it does not exist anymore. Thank you!